File: Server/Models/Dtos/Event/CreateEventDto.cs
namespace Karakatsiya.Models.Dtos.Event
{
    public record CreateEventDto(
        string Title,
        string Description,
        DateTime StartDate,
        string LocationName,
        string City,
        string Street,
        string? HouseNumber,
        double? Latitude,
        double? Longitude,
        string? OsmId,
        string? ExternalTicketUrl,
        string? ContactLinks
    );
}

File: Client/src/app/features/events/dtos/create-event.dto.ts
import { CreateEventPhotoDto } from "./create-event-photo.dto";

export interface CreateEventDto {
  title: string;
  description: string;
  startDate: string;
  locationName: string;
  city: string;
  street: string;
  categoryId: string;
  performers: string[];
  houseNumber?: string;
  latitude?: number;
  longitude?: number;
  osmId?: string;
  externalTicketUrl?: string;
  contactLinks?: string;
  isVip?: boolean;
  photos: CreateEventPhotoDto[];
}